pkgs/applications/networking/ipfs/default.nix

```diff
@@ -2,9 +2,11 @@
 
 buildGoModule rec {
   pname = "ipfs";
-  version = "0.11.0";
+  version = "0.11.0"; # When updating, also check if the repo version changed and adjust repoVersion below
   rev = "v${version}";
 
+  repoVersion = "11"; # Also update ipfs-migrator when changing the repo version
```

Comment:
Is this always the minor version or just coincidence?

Reply:
Just a coincidence, read the commit message or the PR description.